File: level_1/PYTHON/하샤드 수.py
'''
def solution(x):
    answer = True
    sum=0
    for i in str(x):
        sum+=int(i)
    if x%sum!=0:
        answer=False
    return answer
    '''
def solution(n):
    return n % sum([int(c) for c in str(n)]) == 0

File: level_1/C++/하샤드 수.cpp
#include <vector>

using namespace std;

bool solution(int x) {
	int sum = 0;
	int no = x;
	while (no>0) {
		sum += (no % 10);
		no /= 10;
	}
	return x%sum == 0 ? true : false;
}